stdlib/public/core/ContiguousArray.swift

```diff
@@ -932,6 +932,8 @@ extension ContiguousArray: RangeReplaceableCollection {
   }
 }
 
+extension ContiguousArray : ContiguousCollection {}
```

Comment:
should `EmptyCollection` be `ContiguousCollection`?

Reply:
It likely can't hurt, especially if we're doing this for `CollectionOfOne`.